build_and_push.sh

```diff
@@ -6,8 +6,23 @@ set -o pipefail
 IMG="lidofinance/oracle:latest"
 export DOCKER_CONFIG=$HOME/.lidofinance
 
+#### Version info ####
+if ! python ./helpers/get_git_info.py > ./version.json; then
+  echo "python script error"
+  exit 1
+fi
+
+#### Build and push ####
+
 echo "Building oracle Docker image..."
-docker build -t $IMG .
+docker build \
+  --build-arg VERSION="$(cat ./version.json | jq .version)" \
+  --build-arg MESSAGE="$(cat ./version.json | jq .message)" \
+  --build-arg DATETIME="$(cat ./version.json | jq .datetime)" \
+  --build-arg TAGS="$(cat ./version.json | jq .tags)" \
+  --build-arg BRANCH="$(cat ./version.json | jq .branch)" \
+  -t $IMG \
+  .
 
 echo "Pushing image to the Docker Hub"
```

Comment:
Since the most of users will run the script without ability to push, it should be renamed to just 'build.sh'.

Script should stilly try to push, but if access denied, this should be treated gracefully.

The script should output the info about the created image (especially location) and hint how to run it.

extra quotes in labels of docker inspect
```
            "Labels": {
                "BRANCH": "\"issue-55-write-version-to-docker-and-file\"",
                "DATETIME": "\"2020-12-07 18:12:36 +0300\"",
                "MESSAGE": "\"version bumber #55\"",
                "TAGS": "\"None\"",
                "VERSION": "\"v0.1.0-rc.1-3295e13\""
            }
```

if tags are absent, they should be JSON-encoded either as `"tags": null` or `"tags": "".` In docker inspect this should be either `null` or `""`.

In logs the absence should be displayed in any format (None is ok, "" is also good)